src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -213,6 +220,7 @@ const RenderListFeed = (props) => {
             isSelf={item.anonimity ? false : userId === item?.actor?.id}
             onPressScore={showScoreAlertDialog}
             showScoreButton={showScoreButton}
+            isShowDM
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isShowDM` prop is added without a value. Make sure to pass a valid boolean value to this prop based on your application logic.


<!-- This is an auto-generated comment by OSS CodeRabbit -->